src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.ShooterConstants;
+
+public class ShooterSubsystem extends SubsystemBase {
+    private WPI_VictorSPX shooterMotor1 = new WPI_VictorSPX(ShooterConstants.kShooterMotor1Port);
+    private WPI_VictorSPX shooterMotor2 = new WPI_VictorSPX(ShooterConstants.kShooterMotor2Port);
+
+
+  public ShooterSubsystem() {
+    shooterMotor1.setInverted(true);
```

Comment:
-shooter motorları neden inverted?

Reply:
Tam hatırlamıyorum ama Vatan abi atölyedeyken benim motorlarda bir sıkıntı çıkmıştı. Vatan abi setInverted yaptırtmıştı. Daha güvenli gibisinden bir şeyler demişti ondan bende bu motorlara ekledim. Normalde kararsız kalmıştım acaba o ana özelmiydi diye ama 2020 robotunda da görünce ekledim.

Reply:
Hacı ben safetyEnabled için dedim onu, setInverted ne yapıyor kral?

Reply:
Karıştırdım ikisini. setInverted ters bağlamayı kontrol amaçlıydı. Terim karışıklığı oldu:D